ispyb/sp/xtalimaging.py

```diff
@@ -0,0 +1,77 @@
+# xtalimaging.py
+#
+#    Copyright (C) 2020 Diamond Light Source, Karl Levik
+#
+# 2020-01-22
+#
```

Comment:
instead of this comment preamble I'd suggest a
`from __future__ import absolute_import, division, print_function`